kafka/consumer/main.go

```diff
@@ -54,15 +48,19 @@ func main() {
 
 	signals := make(chan os.Signal, 1)
 	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
-	cg, _ := inClusterKafkaConfig()
+
+	brokerList := []string{getEnv(eventHubsBrokerEnvVar)}
+	log.Println("Event Hubs broker", brokerList)
+	consumerGroupID := getEnv(consumerGroupIDEnvVar)
+	log.Println("Sarama client consumer group ID", consumerGroupID)
 
 	t, err := template.ParseFiles("/webtemplates/index.html")
 	if err != nil {
 		log.Printf("Error parsing templates: %s", err.Error())
 		os.Exit(1)
 	}
 
-	consumerGroup, err := sarama.NewConsumerGroup([]string{brokers}, consumergroup, cg)
+	consumerGroup, err := sarama.NewConsumerGroup(brokerList, consumerGroupID, getConfig())
```

Comment:
Does it make sense to use https://github.com/Azure/azure-event-hubs-go instead of sarama?

Reply:
https://github.com/Azure/azure-event-hubs-go  also relies on Sarama.

Reply:
I tried this before using azure-event-hubs-for-kafka. I think both ways work. We still need those credentials and etc.